experimental/orchestrator/docs/Overview.md

```diff
@@ -0,0 +1,109 @@
+# Technical Overview
+
+The Orchestrator is a replacement of the [Bot Framework Dispatcher][1] used in chat bots since 2018. It makes the state of the art natural language understanding methods available to bot developers while at the same time making the process of language modeling quick, not requiring the expertise in [Deep Neural Networks (DNN), Transformers][5], or [Natural Language Processing (NLP)][6]. This work is co-authored with the industry experts in the field and includes some of the top methods used in the [General Language Understanding Evaluation (GLUE)][7] leaderboard. Orchestrator will continue to evolve and adopt the latest advancements in science and in the industry.
+
+Orchestrator enables composability of bots allowing reuse of skills or the entire bots contributed by the community in an easy way without requiring time consuming retraining of the language models. It is our goal to support the community and continue responding to the provided feedback.
+
+## Design Objectives and Highlights
+
+Thanks to the community feedback we compiled a list of objectives and requirements which are addressed in the initial release of the Orchestrator. The [Roadmap](###roadmap) section describes the additional work planned in the upcoming releases. 
+
+#### No [ML][12] or [NLP][6] expertise required

```

Comment:
Personal style suggestion, I would use `###` as opposed to `####` as three stand out better, and better indicate the importance of each section.

E.g.

### No [ML][12] or [NLP][6] expertise required

vs 

#### No [ML][12] or [NLP][6] expertise required

[12]: https://github.com/microsoft/BotBuilder-Samples/pull/2674
[6]: https://github.com/microsoft/BotBuilder-Samples/pull/2674

Reply:
Updated.